src/App.js

```diff
@@ -1,6 +1,4 @@
 import './index.css';
-// import 'bootstrap/dist/css/bootstrap.min.css';
-// import { Button } from 'react-bootstrap';
 import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] The `App.js` and `calculator.js` components should be class-based. kindly fix this

App.js
![image](https://user-images.githubusercontent.com/81932478/218544180-96420a43-b79d-4820-8d1c-02e8e7b3cbb5.png)

calculator.js
![image](https://user-images.githubusercontent.com/81932478/218544329-fa648032-e1f1-4722-a24d-db3a19f9c14e.png)